src/read_config.c

```diff
@@ -18,7 +18,13 @@ along with mfaktc (mfakto).  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include <stdio.h>
+#ifdef _WIN32
 #include <string.h>
```

Comment:
`string.h` should be included unconditionally, it's needed for `strncpy`.

Reply:
My understanding was that strings.h was the equivalent of string.h for windows, but only mainly differed in the naming of the case insensitive string comparison. This seemed better than rolling our own portable string comparison, but perhaps my info was outdated.